src/services/auth/plugins/password/password.schemas.ts

```diff
@@ -7,6 +7,7 @@ import { customType } from '../../../../plugins/typebox';
 import { errorSchemaRef } from '../../../../schemas/global';
 
 export const passwordLogin = {
+  operationId: 'signInWithPassword',
```

Comment:
I think we want to move out of `signIn` `signUp` and instead use `login` and `register` what do you think ?